docs/MigrationGuide.md

```diff
@@ -22,7 +22,11 @@ that will be hidden before parameter's evaluation
 * [#1166](https://github.com/TouK/nussknacker/pull/1166) ```model.conf``` should be renamed to ```defaultModelConfig.conf```
 * [#1218](https://github.com/TouK/nussknacker/pull/1218) FlinkProcessManager is no longer bundled in ui uber-jar. In docker/tgz distribution
 * [#1255](https://github.com/TouK/nussknacker/pull/1255) Moved displaying `Metrics tab` to `customTabs`
-FlinkProcessManager is in different location
+* [#1257](https://github.com/TouK/nussknacker/pull/1257) Improvements: Flink test util package
+    - Moved methods: `runningJobs`, `cancel` (cancelJob) from `MiniClusterExecutionEnvironment` to `FlinkMiniClusterHolder`
+    - Removed `getClusterClient` from `FlinkMiniClusterHolder` interface, because of flink compatibility at Flink 1.9
+    - Added methods: `submitJob`, `listJobs` to `FlinkMiniClusterHolder` 
+    - Renamed `FlinkStreamingProcessRegistrar` to `FlinkProcessManager` 
```

Comment:
I can't see this change in code of this PR. Where it is?

Reply:
It's old change.. I will added this there, because I can't find this `FlinkStreamingProcessRegistrar` on my own integration.